src/renderer/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,53 @@
+import { Dialog as BlueprintDialog } from '@blueprintjs/core'
+import classNames from 'classnames'
+import React from 'react'
+
+import type { DialogProps } from '../dialogs/DialogController'
+
+import styles from './styles.module.scss'
+
+const DEFAULT_WIDTH = 500
+
+type Props = React.PropsWithChildren<
+  {
+    backdropProps?: any
+    canEscapeKeyClose?: boolean
+    canOutsideClickClose?: boolean
+    className?: string
+    fixed?: boolean
+    height?: number
+    width?: number
+  } & Pick<DialogProps, 'isOpen' | 'onClose'>
+>
+
+const Dialog = React.memo<Props>(
+  ({
+    children,
+    canOutsideClickClose = true,
+    canEscapeKeyClose = true,
+    width = DEFAULT_WIDTH,
+    height,
+    ...props
+  }) => {
+    return (
+      <BlueprintDialog
+        isOpen={props.isOpen}
+        onClose={props.onClose}
+        canOutsideClickClose={canOutsideClickClose}
+        canEscapeKeyClose={canEscapeKeyClose}
+        backdropProps={props.backdropProps}
+        className={classNames(styles.dialog, props.className, {
+          [styles.fixed]: props.fixed,
+        })}
+        style={{
+          width: width && `${width}px`,
+          height: height && `${height}px`,
+        }}
+      >
+        {children}
+      </BlueprintDialog>
```

Comment:
This is the only place now where we use blueprint for dialogues. All of this will fit nicely together with the dialog logic refactor in https://github.com/deltachat/deltachat-desktop/pull/3512

Reply:
we should investigate using the `<dialog>` html tag instead.
Also I think we don't need the animations that blueprint comes with, but even if we do, I think they will be easy to replicate.